app/controllers/workers_controller.rb

```diff
@@ -42,11 +36,15 @@ def activate #TODO  use castom validation validates method
   end
 
   def deactivate
-    if @worker.deactivate!
-      render json: ["#{@worker.first_name} #{@worker.last_name} has not finished tickets"]
+    unless @worker.user.is_admin
```

Comment:
if case when we have
`unless
  .....
  else
  ....
  end  
`

better to use 
`if 
  .....
  else
  ....
  end
  `